tests/integration/integration_python_test.go

```diff
@@ -1460,3 +1468,137 @@ func TestConfigGetterOverloads(t *testing.T) {
 	// Run a preview. This will typecheck the program and fail if typechecking has errors.
 	e.RunCommand("pulumi", "preview")
 }
+
+func TestDebuggerAttachPython(t *testing.T) {
```

Comment:
Might be good to add a small doc comment explaining what we're testing here -- it looks like we just spin up a program in debug mode, then tell it to run and wait for it to terminate (using appropriate debug events) -- is that right?

Reply:
Yup, that's right.  It's to make sure the command runs in debug mode and responds to debug commands, and completes successfully after the debugger tells it to.  I'll add a comment.